src/common-lib/log.ts

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-console */
+import debug from 'debug';
+
+export class DebugLogger {
+  log: ReturnType<typeof debug>;
+  error: ReturnType<typeof debug>;
+  constructor(location: string) {
+    // set all locations under our `coordinape` namespace
+    const domain = 'coordinape:' + location;
+    this.log = debug(domain + ':log');
+    // log to stdout
+    this.log.log = console.log;
```

Comment:
hey, sorry i didn't get to reviewing this earlier. what's the purpose of this additional property? it's not sufficient to call `this.log` directly to log to console?

Reply:
ah, found the explanation in https://github.com/debug-js/debug#output-streams

but that also says:
```js
log.log = console.log.bind(console); // don't forget to bind to console!
```

Reply:
It worked without the bind step when I was testing this out. Initially had it, but then I figured out we didn't need it.